requirements.txt

```diff
@@ -16,3 +16,4 @@ numpy
 python-dotenv
 requests
 transitions
+paho-mqtt>=1.2
```

Comment:
Is it necessary to specify larger than? Could it just say `paho-mqtt` and we would always use a version higher than 1.2 in any case?

Reply:
Not sure. It was copied from eq_robot_mqtt_broker. Probably don't need to specify version, since it will be always higher than 1.2